syntax = "proto3";

package protos;
option go_package = "./proto";
import "protos/message.proto";


// 定义交易履约系统服务接口
service Python_B {
  rpc Python_B_1(CommonMessage) returns (CommonMessage) {}
  rpc Python_B_2(CommonMessage) returns (CommonMessage) {}
  rpc Python_B_3(CommonMessage) returns (CommonMessage) {}
}
